src/coreclr/jit/importer.cpp

```diff
@@ -1686,6 +1686,13 @@ GenTreeCall* Compiler::impReadyToRunHelperToTree(CORINFO_RESOLVED_TOKEN* pResolv
 
     op1->setEntryPoint(lookup);
 
+    if (IsStaticHelperEligibleForExpansion(op1))
+    {
+        // Keep class handle attached to the helper call since it's difficult to restore it
+        // from arguments/EntryPoint
```

Comment:
> Keep class handle attached to the helper call since it's difficult to restore it from arguments/EntryPoint

Why is it hard to restore it from the entry point? Isn't it just `((ReadyToRunHelperNode)HandleToObject(gtEntryPoint.handle)).Target`? Still needed for the non-AOT case, however.

Reply:
Exactly, I'm still not sure I want to have it enabled for JIT, but it helped to test it properly since AOT has smaller test suite and doesn't support various jit stress tests - I plan to merge it enabled for jit, play with some TE/microbenchmarks + TC=0 mode and decide + will have SPMI data

Reply:
```suggestion
        // Keep class handle attached to the helper call since it's difficult to restore it.
```